lua/sh-tas-utils/copymodelproperty.lua

```diff
@@ -0,0 +1,18 @@
+
+AddCSLuaFile()
```

Comment:
nit: We're doing this automatically for `sh-tas-utils` and `cl-tas-utils` here: https://github.com/TAServers/gmod-utils/blob/master/lua/autorun/tas_utils_init.lua#L13-L16